packages/generator/src/generator.spec.ts

```diff
@@ -34,6 +35,7 @@ describe('generator', () => {
       const project = await generateProject(
         createOptions({
           inputDir:resolve(oDataServiceSpecs,'v2','API_TEST_SRV'),
+          outputDir: 'common-api-hub-metadata',
```

Comment:
This will fix the `file already exists` error

Reply:
Wouldn't it be easier to enable the overwrite flag?

Reply:
I'm fine with both. Setting output dir allows you to check multiple generated files.